File: project_euler/01/015.go
// Copyright [2017] <dmnsn7@gmail.com>

package main

import "fmt"

const cr = 20
const cc = 20

func getComb(a, b int) int64 {
	comb := int64(1)
	for i := 1; i <= b; i++ {
		comb = comb * int64(a-i+1) / int64(i)
	}

	return comb
}

func main() {
	comb := getComb(cr+cc, cr)

	fmt.Printf("%d\n", comb)
}

File: project_euler/01/015.py
#!/usr/bin/env python3
'''Copyright [2017] <dmnsn7@gmail.com>'''

R = 20
C = 20


def get_comb(num_a, num_b):
    comb = 1
    for i in range(1, num_b + 1):
        comb = comb * (num_a - i + 1) // i

    return comb


def main():
    comb = get_comb(R + C, R)

    print(comb)


if __name__ == '__main__':
    main()
